039-AKSEnterpriseGrade/Student/01-containers.md

```diff
@@ -51,8 +51,8 @@ Note the two links at the bottom of the page in the picture above will not work
 
 These docs might help you achieving these objectives:
 
-- [API image documentation and source code](./Resources/api/README.md)
-- [Web image documentation and source code](./Resources/web/README.md)
+- [API image documentation and source code](https://github.com/erjosito/whoami/tree/master/api)
```

Comment:
@erjosito You have changed the learning resource links for the api & web apps from the two README.md files in your /Student/Resources folders to an external repo that you own.  Will your "whoami" repo be the new source of truth for these apps?  We shouldn't link to external repos unless we know who owns the repo (in this case... that's you :)), or if that location is going to be the source of truth for that code.   If this is the new source of truth, how will you keep the api & web folders in /Student/Resources up to date?

Reply:
@jrzyshr  Undecided... If the external repo is the only alternative to the zip file, I would go for it.

Reply:
@erjosito As noted in my comment above, there is no way to avoid the zip file at the current time (until/unless a better solution replaces DownGit).  I'd rather WTH be the source of truth for any code a hack requires a student to use.